homeassistant/components/electric_kiwi/__init__.py

```diff
@@ -24,6 +24,8 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Electric Kiwi from a config entry."""
+    assert entry.unique_id is not None
```

Comment:
So why do we assert it here if we are going to set it down below

Reply:
I've removed the check further below.

Reply:
Okay so now we do the migrate so in theory this is true, but I also don't see any reason why we need this here